crates/runtime/src/datafusion/query.rs

```diff
@@ -211,11 +212,24 @@ impl Query {
                 t
             });
 
-            let df = DataFrame::new(session, *plan);
+            let physical_plan = match session.create_physical_plan(&plan).await {
+                Ok(stream) => stream,
+                Err(e) => {
+                    let e = find_datafusion_root(e);
+                    let error_code = ErrorCode::from(&e);
+                    handle_error!(
+                        tracker,
+                        &request_context,
+                        error_code,
+                        e,
+                        UnableToExecuteQuery
+                    )
+                }
+            };
```

Comment:
This work is needed to extract `physical_plan`. And once we have it, we don't really need `DataFrame` - so it was completely removed.